xcode/Subconscious/Shared/Components/OmniboxView.swift

```diff
@@ -20,23 +20,29 @@ struct OmniboxView: View {
             HStack(spacing: 0) {
                 Spacer()
                 Text(verbatim: title)
+                    .font(.callout)
                     .foregroundColor(.accentColor)
```

Comment:
It might be nice to come up with an empty state here. imo it looks like the app is in a broken state when opening a new note right now - like it failed to load the UI until it auto-titles after 15 seconds:

<img width="404" alt="image" src="https://user-images.githubusercontent.com/5009316/223882917-0ff5b9a5-fad9-461c-b43a-c1c0cf11f04c.png">

Could do something like this:
```swift
 if title.count == 0 {
    Text(verbatim: "Untitled")
        .font(.callout)
        .foregroundColor(.secondary)
} else {
    Text(verbatim: title)
        .font(.callout)
        .foregroundColor(.accentColor)
}
```

Resulting in: 
<img width="403" alt="image" src="https://user-images.githubusercontent.com/5009316/223883108-54689507-74ca-47df-9789-7fc40e355be5.png">

Reply:
I suggest following up in a subsequent PR https://github.com/subconsciousnetwork/subconscious/issues/437